selfdrive/car/tests/routes.py

```diff
@@ -182,6 +182,7 @@
   CarTestRoute("cd9cff4b0b26c435|2021-05-13--15-12-39", TOYOTA.CHR),
   CarTestRoute("ea8fbe72b96a185c|2023-02-08--15-11-46", TOYOTA.CHR_TSS2),
   CarTestRoute("57858ede0369a261|2021-05-18--20-34-20", TOYOTA.CHRH),
+  CarTestRoute("6719965b0e1d1737|2023-02-07--08-26-18", TOYOTA.CHRH_TSS2),
```

Comment:
Can you put your device on WiFi to upload the logs?

Reply:
It's done. Another one for you, if you need more : 6719965b0e1d1737|2023-02-07--20-48-39--1